ftw/mail/utils.py

```diff
@@ -11,6 +11,7 @@
 IMG_SRC_RE = re.compile(r'<img[^>]*?src="cid:([^"]*)', re.IGNORECASE|re.DOTALL)
 BODY_RE = re.compile(r'<body>(.*)</body>', re.IGNORECASE|re.DOTALL)
 APPLE_PARTIAL_ENCODING_RE = re.compile(r'^"(.*=\?.*\?=.*)"( <.*>)$')
+ENCODED_WORD_WITHOUT_LWSP = re.compile(r'(=\?.*?\?=)(\r\n)')
```

Comment:
The 2nd group `(\r\n)` is not matching linear-whitespace but only the carriage return part of it.
Shouldn't this be `(\r\n[ \t])`, so that it does match `\r\n<SPACE>` or `\r\n\t` as the linear-whitespace is defined?